.storybook/themes/news.scss

```diff
@@ -633,6 +633,15 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				share-bar: (
```

Comment:
Why don't we set all the styles at the mobile breakpoint and then just change the `display` value at the desktop? That way we will not need to specify all `initial` values